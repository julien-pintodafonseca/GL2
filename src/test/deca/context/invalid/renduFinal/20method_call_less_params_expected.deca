// Description:
//      MethodCall : incompatibilité au niveau de la signature de la méthode
//      => Les paramètres passés à la fonction lors de son appel doivent correspondre à sa signature
//
// Resultats:
//      Erreur contextuelle
//      Ligne 21 : La méthode plus prend en entrée 1 paramètre au lieau de 2 paramètres

class A {
    int x;
    int plus(int y){
        y=x+y;
        return y;
    }
}

{
    A a = new A();
    int x = 3;
    float z = 4;
    println("z=", a.plus(x, z));
}
