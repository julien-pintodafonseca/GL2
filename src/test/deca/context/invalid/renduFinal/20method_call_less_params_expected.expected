20method_call_less_params_expected.deca:21:18: La méthode suivante prend en entrée 1 paramètre(s) au lieu de 2 paramètre(s) : plus.
